flexmeasures/data/models/planning/storage.py

```diff
@@ -1030,6 +1034,28 @@ def compute(self, skip_validation: bool = False) -> SchedulerOutputType:
             for sensor in sensors
         }
 
+        flex_model = self.flex_model
+
+        if not isinstance(self.flex_model, list):
+            flex_model["sensor"] = sensors[0]
+            flex_model = [flex_model]
+
+        soc_schedule = {
+            flex_model_d["state_of_charge"]: integrate_time_series(
+                series=ems_schedule[d],
+                initial_stock=soc_at_start[d],
+                up_efficiency=device_constraints[d]["derivative up efficiency"].fillna(
+                    1
+                ),
```

Comment:
Is the `fillna` still necessary if `integrate_time_series` already has a default? If so, could we move the filling into that function instead?